Mage.Tests/src/test/java/org/mage/test/cards/copy/CopySpellTest.java

```diff
@@ -78,7 +78,6 @@ public void ZadaHedronGrinderBoost() {
         // cast boost and copy it for another target (lion will not get boost cause can't be targeted)
         castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "Angelic Blessing", "Zada, Hedron Grinder");
 
-        setStrictChooseMode(true);
```

Comment:
Strict mode is critical fail (e.g. something wrong in the test commands). If you disable it then add comment with the reason.

Reply:
The targeting needs to be fixed anyway, right now it asks the player to select the target even if there's only one option.